modules/rostests/apitests/user32/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ list(APPEND SOURCE
     SetProp.c
     SetScrollInfo.c
     SetScrollRange.c
+    SetWindowPlacement.c
```

Comment:
Can we split the API-test from the implementation in win32ss, into its own PR? I would approve the test-PR immediately without any further discussion, but I do feel unsure about the win32ss-part.

Reply:
Resolved, thank you!

Reply:
#7082